pkg/controller/controller.go

```diff
@@ -1081,47 +1175,117 @@ func (p *csiProvisioner) getPVCSource(ctx context.Context, claim *v1.PersistentV
 	return volumeContentSource, nil
 }
 
+func (p *csiProvisioner) IsGranted(ctx context.Context, claim *v1.PersistentVolumeClaim) (bool, error) {
+	if !utilfeature.DefaultFeatureGate.Enabled(features.CrossNamespaceVolumeDataSource) {
+		return false, nil
+	}
+
+	// Get all ReferenceGrants in snapshot's namespace
```

Comment:
update comment to say "data source" instead of "snapshot"

Reply:
Fixed.